core/domain/email_manager.py

```diff
@@ -2197,132 +2192,166 @@ def send_account_deletion_failed_email(user_id: str, user_email: str) -> None:
+        email_body = email_body_template % (
+            recipient_username, rights_message,
+            category_description)
+
     can_user_receive_email = user_services.get_email_preferences(
         recipient_id).can_receive_email_updates
 
     # Send email only if recipient wants to receive.
     if can_user_receive_email:
-        email_body = email_body_template % (
-            recipient_username, review_category_description,
-            reviewer_rights_message, to_review)
         _send_email(
             recipient_id, feconf.SYSTEM_COMMITTER_ID,
-            feconf.EMAIL_INTENT_ONBOARD_REVIEWER, email_subject, email_body,
+            feconf.EMAIL_INTENT_ONBOARD_CD_USER, email_subject, email_body,
             feconf.NOREPLY_EMAIL_ADDRESS)
 
 
-def send_email_to_removed_contribution_reviewer(
+def send_email_to_removed_cd_user(
     user_id: str,
-    review_category: str,
+    category: str,
     language_code: Optional[str] = None
 ) -> None:
-    """Sends an email to user who is removed from the reviewer position.
+    """Sends an email to user who is removed from a specific 
+        contributor position.
 
     Args:
         user_id: str. The ID of the user.
-        review_category: str. The category which for which review role is
-            removed.
+        category: str. The category in which user can no longer review or 
+            submit contributions.
         language_code: None|str. The language code for a language if the review
-            item is translation or voiceover else None.
+            item is translation else None.
 
     Raises:
-        Exception. The review category is not valid.
+        Exception. The contribution category is not valid.
         Exception. The language_code cannot be None if the review category is
-            'translation' or 'voiceover'.
+            'translation'.
     """
-    if review_category not in REMOVED_REVIEWER_EMAIL_DATA:
-        raise Exception('Invalid review_category: %s' % review_category)
-
-    review_category_data = REMOVED_REVIEWER_EMAIL_DATA[review_category]
-    email_subject = 'You have been unassigned as a %s reviewer' % (
-        review_category_data['review_category'])
+    if category not in REMOVED_CD_USER_EMAIL_DATA:
+        raise Exception('Invalid category: %s' % category)
+
+    category_data = (
+        REMOVED_CD_USER_EMAIL_DATA[category])
+    if category in (
+        [constants.CD_USER_RIGHTS_CATEGORY_SUBMIT_QUESTION]):
+        email_subject = 'You have been unassigned as a %s submitter' % (
+            category_data['category'])
+    else:
+        email_subject = 'You have been unassigned as a %s reviewer' % (
+            category_data['category'])
 
-    if review_category in [
-            constants.CONTRIBUTION_RIGHT_CATEGORY_REVIEW_TRANSLATION,
-            constants.CONTRIBUTION_RIGHT_CATEGORY_REVIEW_VOICEOVER]:
+    if category in [
+            constants.CD_USER_RIGHTS_CATEGORY_REVIEW_TRANSLATION,
```

Comment:
Deindent this by 4, or move it to the end of the previous line / replace it with an equality.

Reply:
Done! Thanks for helping me with the contribution workflow.